docker/README.md

```diff
@@ -55,10 +55,13 @@
 
 ## BigDL in Docker
 
-### The work dir is /opt/work
+### Oracle jdk 1.8.0_152 is installed in /opt/jdk1.8.0_152.
```

Comment:
I think you should list all the packages installed here, including JVM, Spark, Python packages, BigDL, etc. And then you can list the detailed folders below.

Reply:
ok.